collection-scripts/gather

```diff
@@ -1,10 +1,12 @@
 #!/bin/bash
 
+CNV_NAMESPACE=${CNV_NAMESPACE:-openshift-cnv}
```

Comment:
I think that upstream we should avoid this ad set kubevirt-hyperconverged as the default

Reply:
Done